Packs/Jizo_NDR/Integrations/Jizo_NDR/Jizo_NDR_description.md

```diff
@@ -0,0 +1,6 @@
+To access the Jizo NDR service, you need to provide
+
+Your Jizo M server URL. For example: "http://172.30.0.38:9001/api/doc"
+The credentials (Username, Password) as parameters for the integration.
```

Comment:
Can we add more info on creating a service account for this integration + the relevant roles that you need to configure to run this instance?

Reply:
done